vignettes/demo.Rmd

```diff
@@ -297,12 +321,13 @@ This is where {fledge} simplifies the release process by making use of the all t
 ### Bump version for release
 
 We wish to release this package as a patch and so we use `fledge::bump_version()` with the "patch" argument.
+Other values for the arguments are "dev" (default), "minor" and "major".
```

Comment:
Note to self: you might need to bump the version one more time if receiving comments from CRAN demanding changes (not compulsory on CRAN end I think, but recommended?).

Reply:
I tend to release with the same version, with #27 we'd just restart the release process in this case. No need to be too specific here.